tests/default_vars.sh

```diff
@@ -320,6 +320,10 @@ WLCLK_dflt=30
 export WLCLK=$WLCLK_dflt
 export CMP_DATAONLY=false
 
+# Defaults for ufs.configure
+export esmf_logkind="ESMF_LOGKIND_MULTI"
+export DumpFields="false"
```

Comment:
I see an unresolved issue with DumpFields (#1765 ). If this is to be parsed, keep as false until fixed

Reply:
@NickSzapiro-NOAA With the newest ESMF release, we should be able to simplify this part of the FV3 cap, since the newer ESMF allows writing fields on tiles directly.